src/Application/Application.csproj

```diff
@@ -5,6 +5,7 @@
     <IsPackable>False</IsPackable>
     <OutputType>Exe</OutputType>
     <RootNamespace>Symbolica.Application</RootNamespace>
+    <RuntimeIdentifier Condition="'$(RuntimeIdentifier)' == ''">$(NETCoreSdkRuntimeIdentifier)</RuntimeIdentifier>
```

Comment:
I hate this and want to get rid of it. I don't think LLVMSharp is quite packaged correctly but maybe I am misunderstanding something. You know more about all this RID and dotnet build type stuff than me so we should discuss all the various bits once you are happy with everything else in this PR.

Reply:
Hmm I'll have a think about this. I think I'll have to do some digging into how LLVMSharp is packaged as I don't quite get what this is doing just from looking at it. What problem did you hit before you added this?

Reply:
The problem is the good old `Unable to load shared library 'libLLVM' or one of its dependencies.`. The shortest issue that has the workaround is [here](https://github.com/microsoft/LLVMSharp/issues/157). There is an older issue [here](https://github.com/microsoft/LLVMSharp/issues/115), including somebody who may be having my current problem of not being able to get it working on Linux (Windows is fine). There is also a similar issue for ClangSharp but with some more detail [here](https://github.com/microsoft/ClangSharp/issues/118) and this eventually leads to underlying NuGet behaviour [here](https://github.com/NuGet/Home/issues/6083).

Reply:
I was rubbish at searching before. The canonical issue is the one [here](https://github.com/NuGet/Home/issues/10571).

Reply:
It looks like you've solved this for our use case by just including the specific runtime deps explicitly.